tbdex/rfq/rfq.go

```diff
@@ -86,6 +92,146 @@ func (r *RFQ) Verify() error {
 	return nil
 }
 
+// VerifyOfferingRequirements verifies that the RFQ meets the requirements of the provided offering.
+// Specifically this includes the following checks:
+//   - payin method is present in the offering
+//   - payin amount is within the offering or payin method's min and max
+//   - payin details satisfy the offering's required payment details
+//   - payout method is present in the offering
+//   - payout details satisfy the offering's required payment details
+//   - claims satisfy the offering's required claims
+func (rfq *RFQ) VerifyOfferingRequirements(offering _offering.Offering) error {
+	if rfq.Data.OfferingID != offering.Metadata.ID {
+		return fmt.Errorf("rfq's offering id does not match offering used to evaluate rfq")
+	}
+
+	payinAmount, err := decimal.NewFromString(rfq.Data.Payin.Amount)
+	if err != nil {
+		return fmt.Errorf("failed to parse rfq payin amount: %w", err)
+	}
+
+	var selectedPayinMethod *_offering.PayinMethod
+	for _, method := range offering.Data.Payin.Methods {
+
+		if method.Kind == rfq.Data.Payin.Kind {
+			selectedPayinMethod = &method
+			break
+		}
+	}
+
+	if selectedPayinMethod == nil {
+		return errors.New("rfq payin method not found in offering")
+	}
+
+	var min *decimal.Decimal
+	maybeMins := []string{selectedPayinMethod.Min, offering.Data.Payin.Min}
```

Comment:
oh no! i see why you want to get rid of `Payin.min` at the Payin level. it should be for each payment method

Reply:
maybe this wasn't what you were talking about but filed an issue

https://github.com/TBD54566975/tbdex/issues/333